### DISCUSSION TITLE
How to customize the editor name?

### DISCUSSION THREAD

**Discussion by LuckyFroggy** [coder/code-server#6602] [Q&A]
Question:I would like to display the custom editor name. Is there any way to modify it? I would like to replace the code-server global with a different name.
<img width="1792" alt="image" src="https://github.com/coder/code-server/assets/24515428/7d40302b-8806-4116-8c44-48d8297f7608">

**Comment by code-asher** [comment]
There is `--app-name` but I am not sure if it changes the name everywhere.  If not then a workaround would be to edit the name in `lib/vscode/product.json`.
